Sources/App/Views/MarkdownPage.swift

```diff
@@ -26,6 +26,12 @@ class MarkdownPage: PublicPage {
         super.init(path: path)
     }
     
+    init(path: String, html: String) {
+        self.metadata = [:]
+        self.html = html
+        super.init(path: path)
+    }
```

Comment:
IIUC we only need this init for the tests, right? If so, we should place it in an extension in the tests. (If that’s possible, I recall that `init` extensions have some rules attached.)

Reply:
I believe the issue here is that for it to be in an extension we would need to use a `convenience init` which can only use other initialisers in the class (so would need to pass the `markdownFilename` in!)

This would also mean that we need to make html mutable so I can update it within the init since there's no other way to (in)directly modify it.

Would that still be better than this, do you think?

Reply:
Yeah, that's exactly what I thought might prevent making it an extension and the minor convenience - heh - of having it scoped to the tests doesn't really warrant messing with it.